src/tds/unittests/convert.c

```diff
@@ -333,7 +333,7 @@ main(int argc, char **argv)
 				srctype, tds_prtype(srctype), srclen,
 				desttype, tds_prtype(desttype));
 
-			if (result == TDS_CONVERT_NOAVAIL)
+			if (result != TDS_CONVERT_NOAVAIL)
```

Comment:
no, this test is here to make it sure that we support this conversion, as tds_willconvert told us.
not said however that maybe the test could be more strict.

Reply:
I was a bit surprised to find the test not fail when I expected it to.  
Thanks for the explanation that it's not restrictive by design, so I'll take out the patch.

(For what is worth, the test passes for me as such / strict).